pokemongo_bot/__init__.py

```diff
@@ -548,6 +557,7 @@ def _register_events(self):
     def tick(self):
         self.health_record.heartbeat()
         self.cell = self.get_meta_cell()
+        inventory.update_web_inventory()
```

Comment:
I've changed this in a recent commit. Added a cell_worker to perform the function and removed it from init.py. Let me know what you think.